internal/support/notifications/container/config.go

```diff
@@ -1,7 +1,5 @@
 /*******************************************************************************
- * Copyright 2017 Dell Inc.
- * Copyright 2018 Dell Technologies Inc.
- * Copyright (c) 2019 Intel Corporation
```

Comment:
According to the Apache License, derivative works must have original copyrights preserved. Please restore the originals.

Reply:
This change was made by Goland when I moved the file from one directory to another.  It wasn't an intentional change on my part and I missed it when I reviewed the changes prior to creating a PR.

While I don't agree that their remove is a violation of the Apache license, I will manually restore the copyright chain as requested.

Reply:
Totally understand. I just don't want to do anything that might offend any companies or come ANYWHERE NEAR lawyers getting mad.